src/controller/python/OpCredsBinding.cpp

```diff
@@ -149,6 +152,35 @@ class TestCommissioner : public chip::Controller::AutoCommissioner
             return CHIP_NO_ERROR;
         }
 
+        if (report.stageCompleted == chip::Controller::CommissioningStage::kGenerateNOCChain)
+        {
+            if (report.Is<chip::Controller::NocChain>())
+            {
+                auto nocChain = report.Get<chip::Controller::NocChain>();
+                MutableByteSpan rcacSpan(const_cast<uint8_t *>(nocChain.rcac.data()), nocChain.rcac.size());
+
+                chip::ByteSpan rcacByteSpan(rcacSpan.data(), rcacSpan.size());
+
+                // Converting rcac to chip cert format to be decyphered by TLV later in python3
+                chip::Platform::ScopedMemoryBuffer<uint8_t> chipRcac;
+                MutableByteSpan chipRcacSpan;
+                chipRcac.Alloc(Credentials::kMaxCHIPCertLength);
```

Comment:
My apologies so I appear to have been mistaken here. Thank you @cecille for delving into the ScopedMemoryBuffer further as you found and let me know that it appears to not auto-free on destruction. Thus, it could indeed have caused leaks the way I had it coded.
I have changed the above code from using the ScopedMemoryBuffer to using a vector as you mentioned.